src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,260 @@
+"""Use dnstwist to fuzz domain names and cross check with a blocklist."""
+# Standard Python Libraries
+import datetime
+import json
+import pathlib
+import traceback
+
+# Third-Party Libraries
+import dnstwist
+import dshield
+import psycopg2.extras as extras
+import requests
+
+# cisagov Libraries
+from pe_reports import app
+
+from .data.pe_db.db_query_source import (
+    addSubdomain,
+    connect,
+    getDataSource,
+    getSubdomain,
+    org_root_domains,
+    query_orgs_rev,
+)
+
+date = datetime.datetime.now().strftime("%Y-%m-%d")
+LOGGER = app.config["LOGGER"]
+
+
+def checkBlocklist(
+    dom, sub_domain_uid, source_uid, pe_org_uid, perm_list, test_flag=False
+):
+    """Cross reference the dnstwist results with DShield blocklist."""
+    malicious = False
+    attacks = 0
+    reports = 0
+    if "original" in dom["fuzzer"] or "dns_a" not in dom:
+        return None, perm_list
+    else:
+        domain_a = str(dom["dns_a"][0])
+        if domain_a == "!ServFail":
+            return None, perm_list
+        LOGGER.info(domain_a)
+
+        # Check IP in blocklist.de API, if tetsing is true, for testing purposes skip this one as it may go offline
```

Comment:
Fix typo and remove a redundant clause:
```suggestion
        # Check IP in blocklist.de API; if testing is true, skip this one as it may go offline
```

Reply:
commit 864d2f186177bfaad4fdd0eb1f97bf1372bf7775